AdvLoggerPkg/Library/AdvancedLoggerLib/AdvancedLoggerCommon.c

```diff
@@ -40,9 +40,9 @@ AdvancedLoggerMemoryLoggerWrite (
   )
 {
   ADVANCED_LOGGER_INFO              *LoggerInfo;
-  EFI_PHYSICAL_ADDRESS              CurrentBuffer;
-  EFI_PHYSICAL_ADDRESS              NewBuffer;
-  EFI_PHYSICAL_ADDRESS              OldValue;
+  UINT32                            CurrentBuffer;

```

Comment:
nit: I think this functions properly. But as we are doing the offset based operations now, should we also rename these variables?

Reply:
See below comment.